### ISSUE TITLE
CUDNN_STATUS_BAD_PARAM error 

### ISSUE THREAD

**Issue by marziehoghbaie** [Fengdalu/learn-an-effective-lip-reading-model-without-pains#16]
HI there,
I can only run the test function on cpu device. Have some one faced this error. even when I train the model, it works fine in train mode but when it calls the test function on validation set, it return 'CUDNN_STATUS_BAD_PARAM ' error. I searched and one solution was that to .float() to all your tensors. I did try this solution, but I'm still seeing the same error.
Do any one have any idea?

**Comment by marziehoghbaie** [comment]
I used hi again,
I found the solution; I replaced  ` with autocast():` in test function with  `with torch.no_grad():` and now it works fine on GPU.
